### ISSUE TITLE
Disable automatic jumping in PDF

### ISSUE THREAD

**Issue by KronosTheLate** [typst/typst#1312]
### Description

It would be nice to have a toggle in the settings that would turn off automatically jumping in the PDF.

### Use Case

The most dramatic use case is when multiple people are editing the document, in which case the preview is both useless and distracting.

Secondly, I just had a thing where the PDF kept jumping to the spot of my latest change multiple times, despite there being no editing of the code.

Third, the automatic jumping does not do well with a zoomed preview, where it keeps "hugging the left wall" so to speak. The desired behavior is to have it stay centered, so that all the content is readable at 125% zoom. This would be a good band-aid fix for that issue.

**Comment by laurmaedje** [comment]
You can disable it in the View menu.

**Comment by KronosTheLate** [comment]
Right, there is a setting scroll on type there! Awesome, so the feature exists.

But is it not a little misplaced? All other settings under "view" only change looks, not behavior. I feel like this would be more natural as a setting...

**Comment by laurmaedje** [comment]
> But is it not a little misplaced? All other settings under "view" only change looks, not behavior. I feel like this would be more natural as a setting...

Perhaps you're right. I'll discuss it with @reknih.
